docs/datasync/datasync-soft-delete.md

```diff
@@ -182,4 +187,47 @@ type CommentDelta {
 }
 ```
 
-Note that there are is no relationship field in the `CommentDelta` type.
\ No newline at end of file
+Notice that there are is no relationship field in the `CommentDelta` type.
+
+
+## Advanced topics
+
+### `@delta` annotation
+
+The `@datasync` annotation provided by `@graphback/datasync` is a shorthand for two separate annotations: `@delta` and `@versioned`. Vanilla Graphback supports `@versioned` out of the box, see [this](../metadata.md) page.
+
+The `@delta` annotation is only provided by the `@graphback/datasync` package. However, we do not support using this annotation on it's own without `@versioned`.
```

Comment:
This paragraph is good but.. Let's create issue to supply custom function to select field name that will be used for deltas. WDYT?

Reply:
Definitely!